evennia/contrib/utils/name_generator/namegen.py

```diff
@@ -0,0 +1,432 @@
+"""
+# Random Name Generator
+
+Contribution by InspectorCaracal (2022)
+
+A module for generating random names, both real-world and fantasy. Real-world
```

Comment:
Oh! I thought that they were supposed to be the same, since that's how most of the existing contribs look. I'll keep that in mind for the future.

Reply:
The existing contribs mostly looks like this because most contribs used to be a single module (not in a folder) and the requirement of README.md came when I refactored the contrib structure; I then had to use the headers as READMEs since there was no other info. I rewrote most headers of contribs I did myself (but not all, it was a lot of work).